control_panel_api/tests/test_services.py

```diff
@@ -92,6 +97,28 @@ def test_app_delete(self, mock_delete_role):
 
         mock_delete_role.assert_called_with(expected_role_name)
 
+    @patch('control_panel_api.aws.attach_policy_to_role')
+    def test_apps3bucket_create(self, mock_attach_policy_to_role):
+        app = App.objects.create(slug = 'appslug')
```

Comment:
PEP8 - no spaces around `=` in keyword arguments

Reply:
😱 🙀 😱 🙀 😱 🙀 😱 🙀 😱 🙀

Reply:
Changing it 👍 (also interesting that my editor PEP8 thing didn't change it!)